ansible/roles/demo_prep/templates/rds-instance.yml.j2

```diff
@@ -0,0 +1,27 @@
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  name: rds-postgres
+  namespace: "{{ demo_project_name }}"
+spec:
+  serviceClassName: rds-postgres-apb
+  planName: default
+  parameters:
+    aws_access_key: "{{ lookup('env','AWS_ACCESS_KEY_ID') }}"
+    aws_secret_key: "{{ lookup('env','AWS_SECRET_ACCESS_KEY') }}"
+    namespace: "{{ demo_project_name }}"
+    openshift_target: "https://{{ openshift_hostname }}:8443"
+    subnet: "test_awsdemo_rds_group"
+    vpc_security_groups: "sg-dec9b0a1"
```

Comment:
vpc_security_groups: {{ my_security_group_id }}

The security group is the big one we want to lookup since the ID changes.
We are already looking it up and registering as a fact on the host, so we can just use the above fact.
https://github.com/fusor/catasb/blob/master/ansible/reset_environment.yml#L22
https://github.com/fusor/catasb/blob/master/ansible/setup_environment.yml#L22